tv-app/effecticeTV/app/src/main/java/band/effective/office/tv/domain/models/Employee/EmployeeInfoEntity.kt

```diff
@@ -0,0 +1,44 @@
+package band.effective.office.tv.domain.models.Employee
+
+import band.effective.office.tv.utils.DateUtlils
+
+class EmployeeInfoEntity(
+    val firstName: String,
+    val startDate: String,
+    val nextBirthdayDate: String,
+    val photoUrl: String,
+)
+
+fun processEmployeeInfo(data: List<EmployeeInfoEntity>): MutableList<EmployeeInfo> {
+    val resultList = mutableListOf<EmployeeInfo>()
+    data.map {
+        if (DateUtlils.isCelebrationToday(it.nextBirthdayDate)) {
+            resultList.add(
+                Birthday(
+                    it.firstName,
+                    it.photoUrl,
+                )
+            )
+        }
+        if (DateUtlils.isCelebrationToday(it.startDate)) {
+            resultList.add(
+                Anniversary(
+                    it.firstName,
+                    it.photoUrl,
+                    DateUtlils.getYearsWithTheCompany(it.startDate)
+                )
+            )
+        }
+        if (DateUtlils.isNewEmployeeToday(it.startDate)) {
```

Comment:
`DateUtils` must have only logic for processing date objects. 
Business logic methods such as `isCelebrationToday` and `isNewEmployeeToday` should not be here